bundles/org.openhab.binding.nanoleaf/src/main/java/org/openhab/binding/nanoleaf/internal/model/Layout.java

```diff
@@ -110,25 +118,26 @@ public String getLayoutView() {
                 for (int index = 0; index < noofDefinedPanels; index++) {
 
                     if (positionData != null) {
-                        @Nullable
+                        @SuppressWarnings("null")
                         PositionDatum panel = positionData.get(index);
 
-                        if (panel != null && panel.getPosY() == lineY) {
+                        if (panel.getPosY() == lineY) {
                             map.put(panel.getPosX(), panel);
                         }
                     }
                 }
                 lineY -= shiftWidth;
                 for (int x = minx; x <= maxx; x += shiftWidth) {
                     if (map.containsKey(x)) {
-                        @Nullable
                         PositionDatum panel = map.get(x);
-                        view += String.format("%5s ", panel.getPanelId());
+                        @SuppressWarnings("null")
+                        int panelId = panel.getPanelId();
+                        view += String.format("%5s ", panelId);
                     } else {
                         view += "      ";
                     }
                 }
-                view += System.lineSeparator();
+                view += "\n";
```

Comment:
You are more than right on this. I had actually added the lineSeparator quite some time ago BECAUSE of your mentioned Windows-Support and again this must be related that I had to recover my code from several places. Thanks for spotting it.